mint/run/core/aws-sdk-ruby/aws-stub-tests.rb

```diff
@@ -353,21 +405,21 @@ def presignedPostWrapper(bucket_name, file_name, expires_in_sec, max_byte_size,
   # To be addressed. S3 API 'get_bucket_policy' does not work!
   # def getBucketPolicy(bucket_name)
   #   # Returns bucket policy
-  #   return @@s3.bucket(bucket_name).get_bucket_policy
+  #   return s3.bucket(bucket_name).get_bucket_policy
   # rescue => e
   #   raise e
   # end
 
   #
   # Test case methods
   #
-  def listBucketsTest()
+  def listBucketsTest
     # Tests listBuckets api command by creating
     # new buckets from bucket_name_list
 
     # get random bucket names and create list
-    bucket_name1 = get_random_bucket_name()
-    bucket_name2 = get_random_bucket_name()
+    bucket_name1 = random_bucket_name
+    bucket_name2 = random_bucket_name
```

Comment:
These variables seem unnecessary. `bucket_name_list` can be directly initialized as:

```ruby
bucket_name_list = [random_bucket_name, random_bucket_name]
```

Reply:
They are 2 different names since they are randomly generated, i.e. a random number is appended at a base bucket name. So, the code needs to stay.

Reply:
The generated bucket names are not used later anywhere, so directly calling the function `random_bucket_name` when populating `bucket_name_list` should work.